tss-esapi/tests/transient_key_context.rs

```diff
@@ -92,32 +91,6 @@ fn load_bad_sized_key() {
     );
 }
 
-#[test]
```

Comment:
What's the reason for removing some tests in this file?

Reply:
The tests highlighted here were using an operation that I've removed (loading external keys). I guess it's a bit inconsistent that I've only commented out the one below, did that to not lose the actual key values which are a pain to get (though I guess I could just come back to an old commit and copy them from there). I'll remove that one too.